lib/run-compilation.js

```diff
@@ -41,12 +40,12 @@ module.exports = function runCompilation (options, compiler, done) {
 
   compiler.plugin('after-emit', function afterEmit (compilation, callback) {
     if (warnings.length) {
-      compilation.warnings.push(chalk.yellow(options.formatter(warnings)));
+      compilation.warnings.push(new Error(options.formatter(warnings)));
```

Comment:
No, it doesn't break anything, the `Error` wrapper is because webpack is expecting the warnings/errors to be instance of `Error` in order to display them in the compilation errors